Example/NYTPhotoViewer-Swift/ExamplePhoto.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ExamplePhoto.swift
+//  NYTPhotoViewer
+//
+//  Created by Mark Keefe on 3/20/15.
+//  Copyright (c) 2015 The New York Times. All rights reserved.
+//
+
+import UIKit
+
+class ExamplePhoto: NSObject, NYTPhoto {
+
+    var image:UIImage?
+    var placeholderImage:UIImage?
+    let attributedCaptionTitle: NSAttributedString
+    let attributedCaptionSummary = NSAttributedString(string: "summary string", attributes: [NSForegroundColorAttributeName: UIColor.grayColor()])
+    let attributedCaptionCredit = NSAttributedString(string: "credit", attributes: [NSForegroundColorAttributeName: UIColor.darkGrayColor()])
+    
+    init(attributedCaptionTitle: NSAttributedString) {
```

Comment:
I could have this backwards, but isn't the designated initializer generally the one with _more_ arguments? It seems like this one should be the `convenience` initializer while the other one should be designated.